packages/compartment-mapper/src/policy.js

```diff
@@ -28,7 +30,12 @@ export const ATTENUATORS_COMPARTMENT = '<ATTENUATORS>';
  */
 const selectiveCopy = (from, to, list) => {
   if (!list) {
-    list = keys(from);
+    const descs = getOwnPropertyDescriptors(from);
+    list = ownKeys(from).filter(
+      key =>
+        // @ts-expect-error TypeScript still confused about a symbol as index
+        descs[key].enumerable,
+    );
```

Comment:
could use a calming comment:
“by default, copies all enumerable, own, string- and symbol-named properties”

Reply:
Done